conf/cassandra.yaml

```diff
@@ -566,6 +566,48 @@ commitlog_segment_size: 32MiB
 #
 # flush_compression: fast
 
+#
+sstable_compressor:
```

Comment:
@Claudenw this should be `sstable_compression`. We have `commitlog_compression` as well as `hints_compression`. We have even `internode_compression` and `flush_compression`. No reason to have `sstable_compressor` instead of `sstable_compression`.